code/modules/clothing/glasses/hud.dm

```diff
@@ -103,6 +103,20 @@
 		if(report)
 			report.read_paper(user)
 
+/obj/item/clothing/glasses/hud/health/science
+	name = "\improper custom HealthMate HUD" //combined HealthMateHUD and Reagent Scanner HUD for CMO
+	desc = "These HealthMate HUD googles are modified with a light-weight titantium-alloy frame that is custom fitted with extra wiring and low profile components from a reagent analyzer, allowing them to combine the functionality of a HealthMate and reagent scanner HUD without compromising on the form of the googles."
+	icon_state = "healthhud"
+	deactive_state = "degoggles"
+	req_skill = SKILL_MEDICAL && SKILL_RESEARCH
+	req_skill_level = SKILL_MEDICAL_MEDIC && SKILL_RESEARCH_TRAINED
```

Comment:
this will just end up being `req_skill = SKILL_RESEARCH` as the expression is evaluated with the && operator, so you might as well just make it SKILL_RESEARCH and SKILL_RESEARCH_TRAINED. if you *really* wanted this to only be used by people with both, you would have to override `can_use_active_effect(mob/living/carbon/human/user)`, like

```dm

/obj/item/clothing/glasses/hud/health/science/can_use_active_effect(mob/living/carbon/human/user)
	if(skillcheck(user, SKILL_MEDICAL, SKILL_MEDICAL_MEDIC) && skillcheck(user, SKILL_RESEARCH, SKILL_RESEARCH_TRAINED))
		return TRUE

	return FALSE

Reply:
changed it to `req_skill = SKILL_RESEARCH` and `req_skill_level = SKILL_RESEARCH_TRAINED`